ff/handlers/nonbonded.py

```diff
@@ -7,7 +7,8 @@
 import pickle
 
 from rdkit import Chem
-from ff.handlers.utils import match_smirks, sort_tuple
+from ff.handlers.utils import sort_tuple, match_smirks as rd_match_smirks
+from ff.handlers.bcc_aromaticity import match_smirks as oe_match_smirks
```

Comment:
May be better to apply these name changes to the definitions, and move these both into the same file somewhere